migrations/00021_rewards.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE rewards (
+    issuance_week int NOT NULL,
+    connection_streak int,
+    streak_earning int,
+    vehicle_id int NOT NULL CONSTRAINT rewards_vehicle_token_id REFERENCES vehicles (id),
+    aftermarket_token_id int CONSTRAINT rewards_aftermarket_token_id REFERENCES aftermarket_devices (id),
+    aftermarket_earnings int,
```

Comment:
Unfortunately I don’t think `int` (IIRC this is 32 bits) is going to cut it. The numbers are things like 14 x 10^18. We’re probably back to numeric.

I think it is okay to have these token fields be non-null.

Reply:
This is still a problem. For example, look at the transfers from the start of this week:

https://polygonscan.com/tx/0x2298e26e8c50fcf7dcebb656e99f05f5af75d992ae7345602aafd968806d3609#eventlog

14054590318231538246! Very large.

Reply:
This can't be non-null since it will error when any of the tokeniD's isn't present and I try to create any records

Reply:
I mean they can be zero. Nil is fine for now.